src/thor/route_action.cc

```diff
@@ -399,6 +399,7 @@ std::vector<std::vector<thor::PathInfo>> thor_worker_t::get_path(PathAlgorithm*
     float expansion_within_factor = path_algorithm == &bidir_astar ? 2.f : 4.f;
     cost->RelaxHierarchyLimits(relax_factor, expansion_within_factor);
     cost->set_allow_destination_only(true);
+    cost->set_allow_conditional_destination_restrictions(true /* allow */);
```

Comment:
i appreciate that the comment tells the person a little but more about what is going on but we dont follow this convention anywhere else in the code so i dont think we should start at this point

Reply:
I esteem valhalla code convention and I will fix it
but what does valhalla code convention say about methods, classes, structs, variables and enums naming? Why are there so different? :)

Reply:
@dgearhart don't we have a code conventions doc?

Reply:
we do but it recommends googles style guidelines and they are garbage :stuck_out_tongue: